setupTests.js

```diff
@@ -5,6 +5,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+process.env.NODE_ENV = 'test';
```

Comment:
Wouldn't it be better if we set this `ENV` in the package.json command instead of mutating the env on code?